### ISSUE TITLE
Add Keyboard Focus Support

### ISSUE THREAD

**Issue by dm-p** [deneb-viz/deneb#461]
As Deneb currently doesn't have [keyboard focus support](https://learn.microsoft.com/en-us/power-bi/developer/visuals/supportskeyboardfocus-feature) enabled, we lose the focus when we try to navigate the editor with the tab key.

By enabling this, we can cycle the UI and press [Esc] to get back to the Power BI UI. However, we will need to:

- Ensure that focus is restored to the element we were last focused on (or just the editor if it's easier) when we restore focus to the advanced editor
- When we're in the report canvas, we need to be able to correctly show focus on the Vega view, so this will need investigation as to what will work best.
